types/lodash/common/object.d.ts

```diff
@@ -2860,15 +2860,23 @@ declare module "../index" {
          */
         omit<T extends AnyKindOfDictionary>(
             object: T | null | undefined,
-            ...paths: PropertyPath[]
+            ...paths: Array<Many<PropertyName>>
```

Comment:
This type is technically the same as before, but I thought `Many<PropertyName>` better reflected how the value is being used.